microsoft/surface/firmware/surface-go/ath10k/ath10k-replace.nix

```diff
@@ -22,7 +22,7 @@ firmwareLinuxNonfree.overrideAttrs (old: rec {
     cp ${killernetworking_firmware}/board.bin $out/lib/firmware/ath10k/QCA6174/hw3.0/
   '';
 
-  outputHash = "176cf5b9f370x2a532h2afjfkrxy13gqdygc11bam0sg8dlnrv21";
```

Comment:
Fixes #288, @ursi ?

When I tested this, the above source was correct, but the hash after-patching needed to be updated

Reply:
unfortunately not
```
error: hash mismatch in fixed-output derivation '/nix/store/afk3yyi7apxdad0bbwd5fi5wf71m9qvs-microsoft-surface-go-firmware-linux-nonfree-20211027.drv':
         specified: sha256-UD0LzWOiLnlrNeiDyvqWq0FuNe5wjMNaV+hKHZLWzzk=
            got:    sha256-diTvvSrCBY4mDKwEwJkwDmWltYivNsCztf7NrPDXRx0=
```

Reply:
Ah, damn.
Might need another update ...

Reply:
what part of the code is responsible for this nondeterminism?

Reply:
Why does this need an outputHash anyway?